packages/gguf/src/gguf.spec.ts

```diff
@@ -177,4 +178,45 @@ describe("gguf", () => {
 			dtype: GGMLQuantizationType.F16,
 		});
 	});
+
+	it("should parse a v1 file", async () => {
```

Comment:
no need to test a v2 file because v2 is just v3 w/o big endian support, right?

<img width="858" alt="image" src="https://github.com/huggingface/huggingface.js/assets/326577/6246372a-368e-490d-b966-5f83d25e3096">

Reply:
That's what I understood